pkg/controller/issueapi/issue.go

```diff
@@ -254,6 +255,8 @@ func (c *Controller) HandleIssue() http.Handler {
 			return
 		}
 
+		c.recordCapacity(ctx, realm, remaining)
```

Comment:
Yea, so we know their total configured limit and the number remaining. I'm gonna defer to @icco on whether we want to alert on a fixed val (e.g. 10) or a ratio here (e.g. < 10% quota remaining).

Reply:
We wanna alert on percentage, but the metrics should be the actual numbers. So record remaining and total here, and in the alert calculate the percentage

Reply:
Can we have alerting based on multiple metrics? From what I can tell Stackdriver only seemed to consider a single metric, am I wrong?

Reply:
Uhm, my understanding was that we can do math now with Query Notation, but I have not tried

Reply:
Yeah, I can do something like the following to calculate memory utilization in metrics explorer via MQL:

```
{ fetch 'compute.googleapis.com/instance/memory/balloon/ram_used'
; fetch 'compute.googleapis.com/instance/memory/balloon/ram_size' }
| join
| div
```
however, couldn't make something similar work via Terraform alert policy filter, it doesn't seem possible to select multiple metrics by `OR`ring the metric type.

I'll dig further, but in the meantime, added metrics for remaining and issued tokens, which is closer to what we want here.